source/usage-examples/command.txt

```diff
@@ -0,0 +1,48 @@
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+You can run all raw database operations using the
+``MongoDatabase.runCommand()`` method. Call the ``runCommand()`` method
+with your command object on an instance of a database for diagnostic and
+administrative tasks such as fetching server stats or initializing a
+replica set.
+
+.. note::
+    Use the :manual:`mongo shell </reference/mongo-shell>` for
+    administrative tasks instead of the Java Driver whenever possible.
```

Comment:
Question: should this be "Java driver" or is "Java Driver" a specific proper noun? I think it's the former since there isn't just a single one in existence.